src/WebARKitController.js

```diff
@@ -58,10 +58,11 @@ export default class WebARKitController {
       if (this.dataHeap) {
         this.dataHeap.set(data)
         //console.log(this.dataHeap);
+        this.webarkit.initTracking(this.id, this.width, this.height)
```

Comment:
if the initTracking function is here i get:

```
00363f5a:0x1f74b Uncaught (in promise) RuntimeError: memory access out of bounds
    at <anonymous>:wasm-function[446]:0x1f74b
    at <anonymous>:wasm-function[309]:0x131fc
    at <anonymous>:wasm-function[464]:0x22be8
    at <anonymous>:wasm-function[2095]:0xa4bcb
    at <anonymous>:wasm-function[655]:0x3212a
    at <anonymous>:wasm-function[2278]:0xbb87a
    at <anonymous>:wasm-function[915]:0x45990
    at <anonymous>:wasm-function[868]:0x41643
    at U (<anonymous>:wasm-function[1459]:0x759db)
    at A.dynCall_iiiii (http://localhost:3000/dist/WebARKit.js:10:1552615)
```

i would know what is wrong...

Reply:
this is definetely fixed